src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -377,55 +281,38 @@ private async Task<SyntaxToken> RenameAndAnnotateAsync(SyntaxToken token, Syntax
                                 suffix = "EventHandler";
                             }
                         }
-
-                        // This is a conflicting namespace declaration token. Even if the rename results in conflict with this namespace
-                        // conflict is not shown for the namespace so we are tracking this token
-                        if (!isRenameLocation && symbol is INamespaceSymbol && token.GetPreviousToken().IsKind(SyntaxKind.NamespaceKeyword))
-                        {
-                            return newToken;
-                        }
                     }
 
                     // Rename Token
-                    if (isRenameLocation && !this.AnnotateForComplexification)
+                    if (!this.AnnotateForComplexification)
                     {
                         var oldSpan = token.Span;
-                        newToken = RenameToken(token, newToken, prefix, suffix);
+                        newToken = RenameToken(token, newToken, prefix, suffix, isVerbatim, replacementText, originalText, replacementTextValid);
 
                         AddModifiedSpan(oldSpan, newToken.Span);
                     }
 
                     var renameDeclarationLocations = await
                         ConflictResolver.CreateDeclarationLocationAnnotationsAsync(_solution, symbols, _cancellationToken).ConfigureAwait(false);
 
-                    var isNamespaceDeclarationReference = false;
-                    if (isRenameLocation && token.GetPreviousToken().IsKind(SyntaxKind.NamespaceKeyword))
-                    {
-                        isNamespaceDeclarationReference = true;
-                    }
-
+                    var isNamespaceDeclarationReference = token.GetPreviousToken().IsKind(SyntaxKind.NamespaceKeyword);
                     var isMemberGroupReference = _semanticFactsService.IsInsideNameOfExpression(_semanticModel, token.Parent, _cancellationToken);
 
                     var renameAnnotation =
                             new RenameActionAnnotation(
                                 token.Span,
-                                isRenameLocation,
+                                isRenameLocation: true,
                                 prefix,
                                 suffix,
                                 renameDeclarationLocations: renameDeclarationLocations,
-                                isOriginalTextLocation: isOldText,
+                                isOriginalTextLocation: false,
```

Comment:
Why is this always false now?

Reply:
Emm, after more thoughts it seems like we should comparing the replacement text with all the renaming symbols's original name here.
Good catch

Reply:
Ahh, good catch.